tutorials/vercel-setup-guide/index.md

```diff
@@ -0,0 +1,286 @@
+---
+description: Setup guide for configuring Turso using environment variables in an application deployed to Vercel.
+keywords:
+  - turso
+  - tutorial
+  - guide
+  - vercel
+  - environment variable
+---
+
+# Vercel setup guide
+
+In this setup guide, we are deploying a web application made in [Nuxt] that’s
+using [Turso] as its database.
+
+The project being deployed is available on this [GitHub repo].
+
+## Prerequisites
+
+1. [A Vercel account].
+2. The Turso CLI installed in your machine ([installation instructions]).
+
+## 1. Set up the Turso database
+
+### 1a. Create a new database
+
+Run the following CLI command:
+
+```bash
+turso db create topwebframeworks
+```
+
+### 1b. Access the database using the shell.
+
+Run the following CLI command
+
+```bash
+turso db shell topwebframeworks
+```
+
+### 1c. Define and populate the database
+
+Copy and paste the following SQL statements to the shell to create tables,
+indexes, and seed the database with sample data.
+
+```sql
+-- create frameworks table
+create table frameworks (
+    id integer primary key,
+    name varchar (50) not null,
+    language varchar (50) not null,
+    url text not null,
+    stars integer not nulld
+);
+
+-- "name" column unique index
+create unique index idx_frameworks_name ON frameworks (name);                                                                                             
+
+-- "url" column unique index
+create unique index idx_frameworks_url ON frameworks (url);
+
+-- seed some data
+insert into frameworks(name, language, url, stars) values("Vue".js , "JavaScript", "<https://github.com/vuejs/vue>", 203000),
+("React", "JavaScript", "<https://github.com/facebook/react>", 206000),
+("Angular", "TypeScript", "<https://github.com/angular/angular>", 87400),
+("ASP.NET Core", "C#", "<https://github.com/dotnet/aspnetcore>", 31400),
+("Express", "JavaScript", "<https://github.com/expressjs/express>", 60500),
+("Django", "Python", "<https://github.com/django/django>", 69800),
+("Ruby on Rails", "Ruby", "<https://github.com/rails/rails>", 52600),
+("Spring", "Java", "<https://github.com/spring-projects/spring-framework>", 51400),
+("Laravel", "PHP", "<https://github.com/laravel/laravel>", 73100),
+("Flask", "Python", "<https://github.com/pallets/flask>", 62500),
+("Ruby", "Ruby", "<https://github.com/ruby/ruby>", 41000),
+("Symfony", "PHP", "<https://github.com/symfony/symfony>", 28200),
+("CodeIgniter", "PHP", "<https://github.com/bcit-ci/CodeIgniter>", 18200),
+("CakePHP", "PHP", "<https://github.com/cakephp/cakephp>", 8600),
+("Qwik", "TypeScript", "<https://github.com/BuilderIO/qwik>", 16400);
```

Comment:
There's a syntax error here.  I think each of theses rows needs to have its own "insert into" like the first row.